Create preprocess.py

THUzyt21 opened the pull request:
If there are no spaces between sentences in your text file, use this file to generate a new file, which adds spaces between each token.

<!-- Demo: https://github.com/PaddlePaddle/PaddleNLP/pull/26 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->

### PR changes
<!-- One of [ Models | APIs | Docs | Others ] -->

### Describe
<!-- Describe what this PR does -->

THUzyt21 committed:
Create preprocess.py

If there are no spaces between sentences in your text file, use this file to generate a new file, which adds spaces between each token.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleSpeech?pullRequest=2295) <br/>All committers have signed the CLA.

yt605155624 commented:
挪到 local 文件夹里面吧

THUzyt21 committed:
Delete preprocess.py

THUzyt21 committed:
Create preprocess.py

If there are no spaces between sentences in your text file, use this file to generate a new file, which adds spaces between each token.

yt605155624 reviewed (approved):
LGTM
